addons/website/tests/test_iap.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import odoo
+import odoo.tests
+from odoo.addons.iap.tools import iap_tools
+
+@odoo.tests.tagged('external')
```

Comment:
@odoo/rd-runbot Hi, could you confirm that this is enough for the test to be added in the nightly testing suite?
Is there a way to make this test related to the website nightly "tracking"? As we have for the `standalone` tests which when they fail create a red icon on the runbot front page that only the website team can see.
Otherwise we obviously won't notice it if it fail as we don't check the nightly builds.

(Note that the pr was not yet reviewed, but already asking this)

Reply:
I would suggest you to tag it ('website_nightly', '-standard')

Reply:
Already done :wink: https://github.com/odoo/odoo/pull/125005

And I will add ar now :+1: